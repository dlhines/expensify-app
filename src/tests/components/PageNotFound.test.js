import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/PageNotFound';

test('should render PageNotFound correctly', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
});