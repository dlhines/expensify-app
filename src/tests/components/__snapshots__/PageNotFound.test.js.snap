// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PageNotFound correctly 1`] = `
<div>
  404 - NotFoundPage 
  <Link
    replace={false}
    to="/"
  >
    Go To Dashboard
  </Link>
</div>
`;
